### PR TITLE
Add notification priority field to message object (GCM only)

### PR DESCRIPTION
**Pull request by ruslankovalov**

### CODE REVIEW

File: src/Mcfedr/AwsPushBundle/Message/Message.php [hunk 1]
```diff
@@ -608,6 +642,7 @@ private function getGcmJson()
             'collapse_key' => $this->collapseKey,
             'time_to_live' => $this->ttl,
             'delay_while_idle' => $this->delayWhileIdle,
+            'priority' => $this->priority ? $this->priority : self::PRIORITY_HIGH,
```
Comment (mcfedr) [inline src/Mcfedr/AwsPushBundle/Message/Message.php]: Maybe dont add the key unless the user has set it? or set $this->priority to be high by default

Comment (ruslankovalov) [reply src/Mcfedr/AwsPushBundle/Message/Message.php]: this line makes high priority as a default one unless other is set, doen't it? 
or is it better to set the value when I create $priority field e.g.
private $priority = self::PRIORITY_HIGH;
?

Comment (mcfedr) [reply src/Mcfedr/AwsPushBundle/Message/Message.php]: yea, I'd set it in the field and avoid having to deal with `null`

**Comment by mcfedr** [comment]
Released as 6.7.0. Thanks @ruslankovalov

**Pull request merged**